mantidproject/mantid
# Fix editing legends in-situ bug on Linux

**Description of work.**
This PR fixes a bug with editing legends in-situ on Linux platforms. This was due to not using the legend_set_draggable method, which picks the appropriate Matplotlib API to make the legend draggable.


**To test:**
- Plot any data
- Double click on a legend entry to edit it
- This should work


Fixes #30304 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
